content/docs/3.0/tasks/best-practices/pd-scheduling.md

```diff
@@ -0,0 +1,268 @@
+
+`balance-leader` and `balance-region` share a similar scheduling process:
+
+1. Rate stores according to their resource availability.
+2. `balance-leader` or `balance-region` constantly transfer leaders or peers from stores with high scores to those with low scores.
+
+However, their rating methods are different. `balance-leader` uses the sum of all region sizes corresponding to leaders in a store, whereas the way of `balance-region` is relatively complicated. Depending on the specific storage capacity of each node, the rating method of `balance-region` might:
+
+- based on the amount of data when there is sufficient storage (to balance data distribution among nodes).
+- based on the available storage when there is insufficient storage (to balance the storage availability on different nodes).
+- based on the weighted sum of the two factors above when neither of the situations applies.
+
+Because different nodes might differ in performance, you can also set the weight of load balancing for different stores. `leader-weight` and `region-weight` are used to control the leader weight and region weight respectively ("1" by default for both). For example, when the `leader-weight` of a store is set to "2", the number of leaders on the node is about twice as many as that of other nodes after the scheduling stabilizes. Similarly, when the `leader-weight` of a store is set to "0.5", the number of leaders on the node is about half as many as that of other nodes.
+
+### Hot regions scheduling
+
+For hot regions scheduling, use `hot-region-scheduler`. Currently, in TiDB 3.0, the process is performed as follows:
+
+1. Count hot regions by determining read/write traffic that exceeds a certain threshold for a certain period based on the information reported by stores.
+2. Redistribute these regions in a similar way to load balancing.
+
+For hot write regions, `hot-region-scheduler` attempts to redistribute both region peers and leaders; for hot read regions, `hot-region-scheduler` only redistributes region leaders.
+
+### Cluster topology awareness
+
+Cluster topology awareness enables PD to distribute replicas of a region as much as possible. This is how TiKV ensures high availability and disaster recovery capability. PD continuously scans all regions in the background. When PD finds that the distribution of regions is not optimal, it generates an operator to replace peers and redistribute regions.
+
+The component to check region distribution is `replicaChecker`, which is similar to a scheduler except that it cannot be disabled. `replicaChecker` schedules based on the configuration of `location-labels`. For example, `[zone,rack,host]` defines a three-tier topology for a cluster. PD attempts to schedule region peers to different zones first, or to different racks when zones are insufficient (for example, 2 zones for 3 replicas), or to different hosts when racks are insufficient, and so on.
```

Comment:
Suggest we inter-link with the https://tikv.org/docs/3.0/tasks/configure/topology/ article